.github/workflows/docker-cronjob.yml

```diff
@@ -0,0 +1,79 @@
+
+name: Rebuild Docker images for the latest Flower version
+
+on:
+  schedule:
+    # At 01:00 on Monday.
+    - cron: "0 1 * * 1"
+  workflow_dispatch:
+
+jobs:
+  parameters:
+    if: ${{ github.repository == 'adap/flower' }}
+    name: Collect docker build parameters
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    outputs:
+      pip-version: ${{ steps.parameters.outputs.pip-version }}
+      setuptools-version: ${{ steps.parameters.outputs.setuptools-version }}
+      matrix: ${{ steps.parameters.outputs.matrix }}
+    steps:
+      - id: flwr-version
+        run: |
+          FLWR_VERSION=$(pip index versions flwr 2>/dev/null | egrep -o '([0-9]+\.){2}[0-9]+' | head -n 1)
+          echo "flwr-version=$FLWR_VERSION" >> "$GITHUB_OUTPUT"
+
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          # ref: v${{ steps.flwr-version.outputs.flwr-version }}
```

Comment:
Isn't this line a duplicate of the next?

Reply:
good catch! i removed it